config-linux.md

```diff
@@ -169,7 +169,7 @@ In addition to any devices configured with this setting, the runtime MUST also s
 ## <a name="configLinuxControlGroups" />Control groups
 
 Also known as cgroups, they are used to restrict resource usage for a container and handle device access.
-cgroups provide controls (through controllers) to restrict cpu, memory, IO, pids and network for the container.
+cgroups provide controls (through controllers) to restrict cpu, memory, IO, pids, network and rdma resources for the container.
```

Comment:
The kernel docs aren't particularly consistent about this, but since RDMA is [an acronym for remote direct memory access][1], I think we want to use RDMA here (and in other places, except for the literal `rdma`).

[1]: https://en.wikipedia.org/wiki/Remote_direct_memory_access

Reply:
ok. yes, I will change rdma to RDMA.